packages/web-app/src/modules/home-views/components/NavigationBarWithNovuNotifications.tsx

```diff
@@ -1,50 +1,64 @@
-import { useFetchNotifications, useNotifications } from '@novu/notification-center'
+import type { IMessageId } from '@novu/notification-center'
+import { useFetchNotifications, useMarkNotificationsAs } from '@novu/notification-center'
 import { NavigationBar, type NavigationBarProps } from '@saladtechnologies/garden-components'
 import type { FunctionComponent } from 'react'
-import { useEffect, useState } from 'react'
+import { useCallback, useEffect } from 'react'
 import { FeatureFlags, useFeatureManager } from '../../../FeatureManager'
 import { getConfiguredNovuBannerNotifications } from '../../notifications/utils'
 
 export const NavigationBarWithNovuNotifications: FunctionComponent<NavigationBarProps> = (props) => {
-  const novu = useNotifications()
+  const { isNotificationsDrawerOpened, onOpenNotificationsDrawer, onCloseNotificationsDrawer } = props.notifications
+
+  const featureManager = useFeatureManager()
+  const isAchievementsFeatureFlagEnabled = featureManager.isEnabled(FeatureFlags.Achievements)
+
   const fetchNotificationsData = useFetchNotifications({ query: { read: false, limit: 10 } })
   const fetchNotificationsPageData = fetchNotificationsData.data?.pages[0]
   const unreadNovuNotifications = fetchNotificationsPageData?.data
-  const [isDrawerOpened, setIsDrawerOpened] = useState(false)
-  const markFetchedNotificationsAsSeen = novu?.markFetchedNotificationsAsSeen
-  const featureManager = useFeatureManager()
-  const isAchievementsFeatureFlagEnabled = featureManager.isEnabled(FeatureFlags.Achievements)
+  const filteredUnreadNovuNotifications = unreadNovuNotifications?.filter(
+    (unreadNovuNotification) => !unreadNovuNotification.payload.starChefStatus,
+  )
+  const unseenNotificationsIds = filteredUnreadNovuNotifications
+    ?.filter((filteredUnreadNovuNotification) => !filteredUnreadNovuNotification.seen)
+    .map((unseenNovuNotification) => unseenNovuNotification._id)
+  const hasUnseenNotifications = !!(unseenNotificationsIds && unseenNotificationsIds?.length > 0)
+
+  const { markNotificationsAs } = useMarkNotificationsAs()
+  const handleMarkNotificationAs = useCallback(
+    (messageId: IMessageId, seen: boolean, read: boolean) => {
```

Comment:
as we pass array, let's use plural form probably?

```suggestion
    (messageIds: IMessageId, seen: boolean, read: boolean) => {
```

Reply:
It's can be not an array

Reply:
we passing array and it's confusing: the only one id as parameter but we pass an array. 
I know it's novu stuff, but we can make things more clear on our side